added a section about financial considerations (#3)
DutchPHPConference/conference-diversity-and-inclusion: Guideline.md

## Introduction
The purpose of this document is to create a comprehensive overview of things that can help make your conference or event more accessible to everyone, and under-represented groups in particular. It is not meant as a list of requirements for every conference or event. 

> Please read it carefully and you may be surprised about the many things you can do without spending money (for more on the topic see the section "Financial considerations"). Also consider it not “all or nothing”, taking steps that fit your budget is better than not do anything at all.

Reviewer: ```suggestion
Please read it carefully and you may be surprised about the many things you can do without spending money (for more on the topic see the section "Financial considerations"). Also don't consider it “all or nothing”, taking steps that fit your budget is better than not doing anything at all.
```

**Provide sufficient time between talks, events etc.** 
Remember it can take more time to navigate a venue with a mobility device, especially if it is crowded. So consider the suitable time between talks/events for your conference. 

## 15. Financial considerations

Some of the things suggested here cost money: some more, some less. These costs may seem prohibitively high given the tight budget of conferences.

Low conference ticket prices are an important factor to enable inclusion, so simply increasing ticket prices to enable a child day care may not be in the best interests of inclusion efforts. 

> Also note once again, its ok to initially focus on the topics that do not cost money and then later take the next steps.

Reviewer 2: it's ok (missing ')

Here are some concrete ideas to finance diversity and inclusion efforts at your conference:

* Bring costs down by asking for volunteers
    * Maybe someone knows sign language in your community?

>     * Maybe a significant other who is not in IT is willing to help out in the nursery to assist a professional carer? 

Reviewer 3: `professional career`

Reviewer: No, `carer` (someone who cares for children) is correct

>     * Maybe a significant that is not in IT is willing to help out in the children nursery to assist a professional care taker? 

Reviewer: ```suggestion
    * Maybe a significant other who is not in IT is willing to help out in the nursery to assist a professional carer? 
```

Reviewer 4: I'd rather leave out this volunteering in child care as it is not something to take lightly. Also see https://adacamp.org/adacamp-toolkit/childcare/. I will update the childcare paragraph with this link.

> * Setup specific sponsor packages

Reviewer: ```suggestion
* Set up specific sponsor packages
```

* Allow attendees to optionally donate when buying their ticket

>     * Potentially give them a small token of appreciation like a button, a free drink ticket at the conference party ..

Reviewer 2: missing dot in the ellipses at the end

> * Setup an (all year) sponsor fund to collect donations

Reviewer: ```suggestion
* Set up an (all year) sponsor fund to collect donations
```

    * [opencollective.com](https://opencollective.com)
    * [sfconservancy.org](https://sfconservancy.org)

A slightly more out of the box approach could be to save money that usually goes towards speakers. Some of the suggestions can have the bonus effect at getting more people involved in speaking in public which is also great for diversity. Also keep in mind that while conferences are traditionally very focused on having speakers share their knowledge, a very significant (if not even the most significant) aspect of conferences is networking.

* Focus more on regional speakers to reduce travel costs

> * Allow for more user generated content by providing an unconference track with lightning talks (which can even just be about “what do I do at work”)

Reviewer 3: I've encountered `lightning talks` to be talks promoting products and doing advertisement.. not sure it's a good word here? How about `un-conference track with talks which can even just be about “what do I do at work”.`

Author: yeah .. sometimes the term "lightning talk" is "abused" to simply give all the sponsors a slot to promote their products :-/

Author: I have incorporated your suggestion.
